modules/interacting-serverless-apps.adoc

```diff
@@ -24,3 +24,69 @@ helloworld-go   http://helloworld-go.default.example.com   helloworld-go-4wsd2
 $ curl http://helloworld-go.default.example.com
 Hello World: Go Sample v1!
 ----
+
+= Interacting with a serverless application using HTTP2 / gRPC
+
+{product-title} routes do not support HTTP2, and therefore do not support gRPC as this is transported by HTTP2.
+If you use these protocols in your application, you must call the application using the ingressgateway directly.
+
+To to do, you will need to find the ingressgateway's public address and the application's specific host.
+
+.Procedure
+
+. The application's host URL is part of the URL that is surfaced when using `oc get ksvc`, but without the `http://` prefix. For example:
++
+----
+oc get ksvc helloworld-go
+NAME            URL                                                                                 LATESTCREATED         LATESTREADY           READY   REASON
+helloworld-go   http://helloworld-go.default.apps.markusthoemmes20190812.devcluster.openshift.com   helloworld-go-2rjbt   helloworld-go-2rjbt   True
+----
++
+In this case, the application's host URL would be:
++
+----
+helloworld-go.default.apps.markusthoemmes20190812.devcluster.openshift.com
```

Comment:
```suggestion
helloworld-go.default.example.com
```

Reply:
Maybe even just refer to the example above and state that that's how you get the application's host and that for this example the host is "helloworld-go.default.example.com". Maybe we can then omit having the same procedure twice?

Reply:
I didn't even realise, good call, thanks!